crates/config/src/filter.rs

```diff
@@ -56,6 +56,17 @@ impl GlobMatcher {
             return self.matcher.is_match(format!("./{}", path.display()));
         }
 
+        let glob_path = PathBuf::from(self.glob().glob());
```

Comment:
this can be delayed an moved to the && part

we can also use `Path::new` here

Reply:
Good catch, applied the suggested changes